src/backend/funmooc/base/admin.py

```diff
@@ -9,7 +10,7 @@
 from filer.models.virtualitems import FolderRoot
 
 
-class CustomUserAdmin(auth.admin.UserAdmin):
+class CustomUserAdmin(UserAdmin):
```

Comment:
what's the point of this?

Reply:
It's a fix, possibly because a change in last Django version. When i launched the demo-site command it failed on this code arguing there was no UserAdmin in auth.admin.